### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by thisiswhereitype**
Cleaned up throughout, added table of contents, updated support and refreshed examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,44 +2,31 @@
...
-Pangraph offers an api in the module `Pangraph` for accessing of graphs including constructors and getters.  
-You can construct entire graph or edit the results of parser.
-These are imported independently form their own modules:  
-```haskell
-import Pangraph.GraphML.Parser
-```
-
-All parsers currently export the following:
+Pangraph is a Haskell library which offers for parsing and serializations for  
+graph network files. As well as conversions to other Haskell graph formats.
```
Comment (snowleopard) [inline README.md]: graph network files -> graph files ('graph' and 'network' are mostly synonyms).

File: README.md [hunk 2]
```diff
@@ -2,44 +2,31 @@
...
+Some example formats are
+[GraphML](http://graphml.graphdrawing.org/)
+and [Workcraft](https://www.workcraft.org/).
```
Comment (snowleopard) [inline README.md]: Workcraft has a huge website, where exactly should I look for "example formats"?

File: README.md [hunk 3]
```diff
@@ -2,44 +2,31 @@
...
+
+## Contents
+1. [Usage](#usage)  
+2. [Graph File Support](#graph-file-support)
+3. [Library support](#graph-library-support)
+
+## Usage
+Pangraph offers an api in the module `Pangraph` for construction and
```
Comment (snowleopard) [inline README.md]: api -> API

File: README.md [hunk 4]
```diff
@@ -2,44 +2,31 @@
...
+and [Workcraft](https://www.workcraft.org/).
+
+## Contents
+1. [Usage](#usage)  
+2. [Graph File Support](#graph-file-support)
+3. [Library support](#graph-library-support)
+
+## Usage
+Pangraph offers an api in the module `Pangraph` for construction and   
+manipulation of graph networks. The parsers in the library use this module to
```
Comment (snowleopard) [inline README.md]: graph networks -> graphs (do this across the README).

**Comment by snowleopard** [comment]
Thanks, see a couple of minor comments above.

**Comment by thisiswhereitype** [comment]
I have addressed all your comments. I have removed the Workcraft link from the description as it does not have support right now and the directed graph section of the Workcraft website is a link to the Wikipedia page. Thanks for the feedback.

**Comment by snowleopard** [comment]
@thisiswhereitype You use both 'graph' and 'network' terms which may potentially be confusing for the reader who may be wondering: Is there a difference between graphs and networks? If yes, what is it? etc. Since the library is called `pangraph` not `pannetwork` I suggest you stick to 'graphs' throughout the README. Imagine you're a newcomer who is reading the README -- try to get rid of any potential sources of confusion. Maybe add a simple picture of what a graph is (for example, the one from our paper -- with four nodes A-E).

File: README.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
+# pangraph [![Build Status](https://travis-ci.org/tuura/pangraph.svg?branch=master)](https://travis-ci.org/tuura/pangraph)  
 
-[![Build Status](https://travis-ci.org/tuura/pangraph.svg?branch=master)](https://travis-ci.org/tuura/pangraph)  
-
-Pangraph is a Haskell library which offers for parsing and serializations for  
-network files. As well as conversions to other Haskell graph formats. An  
-example format is [GraphML](http://graphml.graphdrawing.org/).
+Pangraph is a Haskell library which offers for parsing and serializations for graph files. As well as conversions to other Haskell graph formats. An example format is [GraphML](http://graphml.graphdrawing.org/). A graphml file
+could for example represent the following graph, with vertices from `A` to `E` and connections, the edges, between them.
+![a-sample-graph](examples/graphs/graph-AE.png?raw=true)
```
Comment (snowleopard) [inline README.md]: @thisiswhereitype Please use a vector (svg) image instead and scale it down -- it's too large at the moment!

**Comment by snowleopard** [comment]
@thisiswhereitype I think you use this file directly: https://github.com/tuura/fdl17-paper/blob/master/fig/network.svg, but you'll need to copy it into this repository.

File: README.md [hunk 6]
```diff
@@ -1,45 +1,30 @@
...
-
-Pangraph offers an api in the module `Pangraph` for accessing of graphs including constructors and getters.  
-You can construct entire graph or edit the results of parser.
-These are imported independently form their own modules:  
+# pangraph [![Build Status](https://travis-ci.org/tuura/pangraph.svg?branch=master)](https://travis-ci.org/tuura/pangraph)  
+
+Pangraph is a Haskell library which offers for parsing and serializations for graph files. As well as conversions to other Haskell graph formats. An example format is [GraphML](http://graphml.graphdrawing.org/). A graphml file
+could for example represent the following graph, with vertices from `A` to `E` and connections, the edges, between them.
+![a-sample-graph](examples/graphs/graph-AE.png?raw=true)
+Source:
```
Comment (snowleopard) [inline README.md]: No need to acknowledge the source of the image in this case since I'm a co-author in both cases.

**Comment by thisiswhereitype** [comment]
I am happy for this to be merged if you also are.

**Comment by snowleopard** [comment]
@thisiswhereitype Thanks, merged!

File: README.md (hunk 2 above)
Comment (benmvp) [reply README.md]: Fixed: the Runtime now rejects observedState without observations, with a dedicated unit test.

**Pull request merged**